### PR TITLE
Add a list of regex in settings.py to remove VM and servers @0x83 from pamela

### PR DESCRIPTION
**Pull request by etnarek**

### CODE REVIEW

File: incubator/settings.py [hunk 1]
```diff
@@ -242,6 +242,10 @@
 OPEN_WEEKDAYS = [0, 1, 2, 3, 4]  # Monday is day 0
 OPEN_HOURS = list(range(7, 23))
 
+IGNORE_LIST_RE = [
```
Comment (C4ptainCrunch) [inline incubator/settings.py]: `import re` ?

File: space/helpers.py [hunk 2]
```diff
@@ -28,6 +28,10 @@ def is_stealth_mode():
     return (not space_may_be_open()) and (not space_is_open(get_redis()))
 
 
+def should_keep(mac):
```
Comment (C4ptainCrunch) [inline space/helpers.py]: Add tests ? :)

File: incubator/settings.py (hunk 1 above)
Comment (etnarek) [reply incubator/settings.py]: oups :p

File: space/helpers.py (hunk 2 above)
Comment (etnarek) [reply space/helpers.py]: done in 011de06fa078c3baa22f33b91e75143e31158df1

**Pull request merged**